Cargo.toml

```
@@ -92,3 +92,4 @@ v2020_7 = ["v2020_4", "ffi/v2020_7"]
 v2020_8 = ["v2020_7", "ffi/v2020_8"]
 v2021_1 = ["v2020_8", "ffi/v2021_1"]
 v2021_2 = ["v2020_1", "ffi/v2021_2"]
+v2021_3 = ["v2020_2", "ffi/v2021_3"]
```

Shouldn't these be

```suggestion
v2021_2 = ["v2021_1", "ffi/v2021_2"]
v2021_3 = ["v2021_2", "ffi/v2021_3"]
```

?

---

Haha yep...hooray for code review (and your eyes!)